### PR TITLE
walletconnect: whitelabel MVP

### PR DESCRIPTION
**Pull request by mPaella**
## Description
api surface

```
 <CrossmintWalletConnect
            walletConnect={{
                projectId: process.env.NEXT_PUBLIC_CROSSMINT_WALLETCONNECT_PROJECT_ID!,
                metadata: {
                    name: "Acme Inc",
                    description: "Acme Web3 Wallets",
                    url: "https://acmemarkets.com",
                    icons: ["https://i.ibb.co/bL0wGsQ/35bec612-6a65-4619-9378-8234466ff60f.webp"],
                },
            }}
            wallets={[fromJsonRpcSigner(signer, primaryWallet!.address)]}
            uiConfig={{
                colors: {
                    background: "#ffebee",
                    backgroundSecondary: "#ffffff",
                    accent: "#f53855",
                    textLink: "#f53855",
                },
            }}
        />
```

## Test plan

1.) built the sdk package
2.) linked the package to crossmint-main
3.) imported WC component from package, and used
image
image
image

japanese
image

### CODE REVIEW

File: packages/client/wallets/walletconnect/package.json [hunk 1]
```diff
@@ -0,0 +1,59 @@
+{
+    "author": "Paella Labs Inc",
```
Comment (alfonso-paella) [inline packages/client/wallets/walletconnect/package.json:2]: Can we start using Paella Inc..

File: packages/client/wallets/walletconnect/src/components/EnterURI/EnterURIInput.tsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+                    value={uri}
+                    disabled={loading}
+                    onChange={(e) => setUri(e.target.value)}
+                />
+
+                <button
+                    className="flex items-center justify-center px-4 py-2 text-sm font-semibold tracking-tight transition-opacity duration-200 rounded-md h-9 hover:opacity-70 disabled:opacity-50"
+                    style={{
+                        backgroundColor: uiConfig.colors?.accent,
+                        color: "white", // TODO: use uiConfig
```
Comment (alfonso-paella) [inline packages/client/wallets/walletconnect/src/components/EnterURI/EnterURIInput.tsx:48]: TODO here

File: packages/client/wallets/walletconnect/src/components/EnterURI/NumberedStep.tsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    const mobile = "w-[1.5rem] h-[1.5rem] w-8 h-8 mr-2";
+    const desktop = "sm:w-12 sm:h-12 sm:mr-4 sm:w-[2rem] sm:h-[2rem] sm:mr-4";
+    const className = classNames(mobile, desktop, "rounded-full flex items-center justify-center flex-none");
+
+    return (
+        <div className="flex items-center">
+            <div
+                className={className}
+                style={{
+                    border: `1px solid ${uiConfig.colors?.accent}`,
```
Comment (alfonso-paella) [inline packages/client/wallets/walletconnect/src/components/EnterURI/NumberedStep.tsx:17]: I dont think colors can be undefined

File: packages/client/wallets/walletconnect/src/components/common/WC_HACK_Modal.tsx [hunk 4]
```diff
@@ -0,0 +1,63 @@
+import { classNames } from "@/utils/classNames";
```
Comment (alfonso-paella) [inline packages/client/wallets/walletconnect/src/components/common/WC_HACK_Modal.tsx:1]: Is there a better name for this class...

File: packages/client/wallets/walletconnect/src/hooks/useModalView.tsx [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+    //         state: { proposal },
+    //     });
+    // }, []);
+
+    // useEffect(() => {
+    //     if (!provider) {
+    //         return;
+    //     }
+    //     provider.on("session_proposal", onSessionProposal);
+    // }, [provider, onSessionProposal]);
```
Comment (alfonso-paella) [inline packages/client/wallets/walletconnect/src/hooks/useModalView.tsx:67]: ?

**Review by alfonso-paella — CHANGES_REQUESTED** [review]
Can you add a test plan?

e.g. I have a test app <where>

I have manually run these test cases:

1.
2.
3. 
...

File: packages/client/wallets/walletconnect/src/components/common/WC_HACK_Modal.tsx (hunk 4 above)
Comment (mPaella) [reply packages/client/wallets/walletconnect/src/components/common/WC_HACK_Modal.tsx:1]: nah, this is very common

**Comment by mPaella** [comment]
im going to merge this in order to proceed, if there are comments i will address later

**Pull request merged**